src/matches/matches.service.ts

```diff
@@ -23,7 +25,9 @@ export class MatchesService {
       refereeId: dto.refereeId,
       observerId: dto.observerId,
       leagueId: leagueId,
+      observerSmsId: await this.sendSMS(dto.matchDate, this.getUserReadableKey(dto.matchDate, leagueIdx, homeTeamIdx), phoneNumber),
```

Comment:
previous way of doing this **not** inline was ok, just type is as string so go:
const observersSmsId: string = await this.sendSms(...)

const match: Match = ... {
...
observerSmsId: observersSmsId,
...
}